addons/mail/static/src/components/composer/composer.xml

```diff
@@ -20,20 +20,36 @@
                         t-ref="dropzone"
                     />
                 </t>
-                <FileUploader
-                    attachmentLocalIds="composer.attachments.map(attachment => attachment.localId)"
-                    isPending="true"
-                    newAttachmentExtraData="newAttachmentExtraData"
-                    threadLocalId="composer.thread.localId"
-                    t-ref="fileUploader"
-                />
+                <t t-if="composer.thread">
+                    <FileUploader
+                        attachmentLocalIds="composer.attachments.map(attachment => attachment.localId)"
+                        isPending="true"
+                        newAttachmentExtraData="newAttachmentExtraData"
+                        threadLocalId="composer.thread.localId"
+                        t-ref="fileUploader"
+                    />
+                </t>
+                <t t-if="composer.messageInEditing">
+                    <FileUploader
+                        attachmentLocalIds="composer.attachments.map(attachment => attachment.localId)"
+                        isPending="true"
+                        newAttachmentExtraData="newAttachmentExtraData"
+                        threadLocalId="composer.messageInEditing.originThread.localId"
+                        t-ref="fileUploader"
+                    />
+                </t>
```

Comment:
2 fileuploader at the same time with same ref... I think it's buggy

Reply:
Solution: mutual exclusion based on `messageInEditing` being truthy or falsy.
Could even avoid code duplication by moving conditional in the props.